src/days-2-4-adv/adv.py

```diff
@@ -49,3 +50,59 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+print (f"""
+    {me.startRoom.name}:
+    {me.startRoom.description}
+    """)
+while True:
+    cmd = input("which way?-> ")
+    if cmd == "q":
+        print ("""
+    Quitting
+        """)
+        break
+    if cmd == "n":
```

Comment:
This works perfectly.  You might consider seeing if you can maybe make this section a little dryer.